projects/fsastorefrontlib/src/cms-components/my-account/order/order-details/order-summary/order-summary.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component, Input } from '@angular/core';
```

Comment:
Since this component has only 1 property coming from input, didn't see any benefit in extending Spartacus' order-summary.component